### PR TITLE
CB-10652 Added the instruction on how to prepare a slave for Appium t…

### PR DESCRIPTION
**Pull request by alsorokin**
…ests

https://issues.apache.org/jira/browse/CB-10652

### CODE REVIEW

**Comment by alsorokin** [comment]
@dblotsky please take a look

File: SLAVES.md [hunk 1]
```diff
@@ -27,6 +27,17 @@ Two special NPM packages are also required for Medic builds on iOS: `ios-sim` an
 
     npm install -g ios-deploy ios-sim
 
+To run Appium tests on real iOS devices you may need to install ios-webkit-debug-proxy using the command:
+
+ ``` center
+ # The first command is only required if you don't have brew installed.
+ > ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```
Comment (dblotsky) [inline SLAVES.md]: Please point the docs to the official Homebrew installation page, since this command might change.

**Comment by dblotsky** [comment]
Will the iOS steps always be necessary, or will they become unneeded later? For Android, do you have a link to the bug that when fixed will obsolete this Android setup?

**Comment by alsorokin** [comment]
@dblotsky iOS steps will always be necessary only if you want to run the tests on real device.
You don't need to run these steps to run the tests on the simulator.

**Comment by alsorokin** [comment]
Also, added the links to the Appium issues describing the problem and the workaround for the chromedriver issue.

File: SLAVES.md [hunk 2]
```diff
@@ -38,6 +49,12 @@ To make the Android commands available on the command line, set the following en
 - `ANDROID_HOME`, equal to the absolute path to the Android SDK directory
 - `PATH`, extended to contain `ANDROID_HOME/tools` and `ANDROID_HOME/platform_tools`
 
+To run Appium tests, you may also need to download the [latest chromedriver](https://sites.google.com/a/chromium.org/chromedriver/downloads) and set the following environment variable:
+
+- `CHROMEDRIVER_EXECUTABLE`, equal to the absolute path to the chromedriver executable
+
+This is needed to avoid the bug in older chromedriver versions that was preventing Appium tests to switch contexts. There are a bunch of github issues created for this bug in Appium project: [1](https://github.com/appium/appium/issues/4429), [2](https://github.com/appium/appium/issues/5526), [3](https://github.com/appium/appium/issues/5616).
```
Comment (dblotsky) [inline SLAVES.md]: Please make the links reference links.

**Pull request merged**